fractions/microprofile/microprofile-restclient/README.adoc

```diff
@@ -0,0 +1,42 @@
+= MicroProfile Rest Client
+
+== CDI Interceptors Support
+
+In general, Rest Client proxies are not created by the CDI container and therefore method invocations do not pass through CDI interceptors.
+In Thorntail, however, you can associate business method interceptors (denoted by `@AroundInvoke` annotation) with a Rest Client proxy.
+You can do so by using interceptor bindings.
```

Comment:
Is there a reason this is a separate sentence, instead of writing _with a Rest Client proxy by using interceptor bindings._ ? In docs, we try to avoid pronouns and references (_do so, do that_ etc) as much as possible because it may not always be clear what we are referring to, and it's not minimalist (TL;DR every word in the docs is there because it serves a purpose).

Reply:
Well, I find it more readable. First, we introduce the feature and then we specify how to use it. But I'm ok with minimalistic approach.